modules/social_features/social_event/modules/social_event_invite/config/install/social_event_invite.settings.yml

```diff
@@ -0,0 +1,10 @@
+invite_enroll: true
+invite_subject: '[user:name] has invited you to join the event [node:title] on [site:name]'
```

Comment:
should this be display name instead? This will ensure correct fallbacks for example when users are hiding their name

$tests['[user:name]'] = $account->getAccountName();
    $tests['[user:account-name]'] = $account->getAccountName();
    $tests['[user:display-name]'] = $account->getDisplayName();